tests/test_non_ascii_paths/code/makefile

```diff
@@ -0,0 +1,8 @@
+all: c
+	mkdir code
+	cd code && python -c "import os; os.mkdir('\x8d\xae\xa2\xa0\xef')"
+	cd code && python -c "open('\x8d\xae\xa2\xa0\xef/\xaf\xa0\xaf\xaa\xa0', 'w').write('Hello!\n')"
+c: 第一.c
```

Comment:
I bet we're also going to test various versions of `make` pretty effectively. ;-)